build/docker/build-tools/build-tools.dockerfile

```diff
@@ -0,0 +1,22 @@
+# As a basic image for building various components of KubeEdge
+FROM ubuntu:18.04
+ARG ARCH=amd64
+RUN apt-get update && apt-get install -y wget \
+    vim git make gcc upx-ucl 
+
+RUN if [ "${ARCH}" = "amd64" ]; then apt-get install -y gcc-aarch64-linux-gnu libc6-dev-arm64-cross gcc-arm-linux-gnueabi libc6-dev-armel-cross ; fi
```

Comment:
For arm arch, it's no needed to install these cross-build pkgs. Install them in amd64 is because we may use it to do the cross-build in future.